src/olympia/git/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from olympia.addons.models import Addon
+from olympia.amo.models import ModelBase
+
+
+class AddonGitExtraction(ModelBase):
+    """
+    This is an add-on related model that stores information related to the git
+    extraction (for code-manager).
+    """
+
+    addon = models.OneToOneField(
```

Comment:
isn't git extraction a thing for each new version?  If so you'd need the fk to `Version` or `File` instead, right?

Reply:
Git extraction extracts each version of an add-on but we want to lock the full add-on to prevent issues (like wrong order of the versions in git or concurrent extraction).